### PR TITLE
Feature/refactor structure

### PR DESCRIPTION
**Pull request by arturminchukov**

### CODE REVIEW

File: application/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const services = require('./services');
+
+module.exports = {
+    services,
```
Comment (BSiarhei) [inline application/index.js]: comma

File: application/services/beerService.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const {beerRepository} = require('../../dataAccess/repository');
+
+class BeerService {
+    getBeers(query) {
```
Comment (BSiarhei) [inline application/services/beerService.js]: query?

File: dataAccess/repository/beerRepository.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const apiUrl = 'https://api.punkapi.com/v2/beers';
```
Comment (BSiarhei) [inline dataAccess/repository/beerRepository.js]: config

File: dataAccess/repository/beerRepository.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const axios = require('axios');
+
+class BeerRepository {
```
Comment (BSiarhei) [inline dataAccess/repository/beerRepository.js]: error handling

File: dataAccess/repository/beerRepository.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+    getBeers(page = 1, itemsPerPage = 10) {
+        return this.requestResponse(`?page=${page}&per_page=${itemsPerPage}`);
+    }
+
+    getBeerById(id) {
+        return this.requestResponse(`/${id}`);
+    }
+
+    getBeersByQuery(beerName) {
```
Comment (BSiarhei) [inline dataAccess/repository/beerRepository.js]: name

File: dataAccess/repository/index.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+const beerRepository = require('./beerRepository');
+
+module.exports = {
+    beerRepository,
```
Comment (BSiarhei) [inline dataAccess/repository/index.js]: comma

File: web/controllers/beerController.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+const {beerService} = require('../../application').services;
+
+class BeerController {
+    async handleBeers(req, res) {
```
Comment (BSiarhei) [inline web/controllers/beerController.js]: name

File: web/server.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+        res.header('Access-Control-Allow-Origin', '*');
+        res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+        next();
+    });
+    app.use('/beers', beersRouter);
+
+    return app;
+};
+
+module.exports = configureServer;
```
Comment (BSiarhei) [inline web/server.js]: app

**Comment by arturminchukov** [comment]
These commits i migration-user branch

**Pull request closed without merging**